exp/controllers/azuremanagedcluster_controller.go

```diff
@@ -86,7 +86,7 @@ func (amcr *AzureManagedClusterReconciler) SetupWithManager(ctx context.Context,
 	// Add a watch on clusterv1.Cluster object for unpause notifications.
 	if err = c.Watch(
 		&source.Kind{Type: &clusterv1.Cluster{}},
-		handler.EnqueueRequestsFromMapFunc(util.ClusterToInfrastructureMapFunc(infrav1exp.GroupVersion.WithKind("AzureManagedCluster"))),
+		handler.EnqueueRequestsFromMapFunc(util.ClusterToInfrastructureMapFunc(ctx, infrav1exp.GroupVersion.WithKind("AzureManagedCluster"), mgr.GetClient(), &infrav1exp.AzureManagedCluster{})),
```

Comment:
Just curious, what changed to the `util.ClusterToInfrastructureMapFunc()` to require multiple args now?

Reply:
here is the PR https://github.com/kubernetes-sigs/cluster-api/pull/6070